pybikes/data/cocacola.json

```diff
@@ -0,0 +1,33 @@
+{
+    "instances": [
+        {
+            "tag": "cork",
+            "meta": {
+                "latitude": 51.893604,
+                "longitude": -8.494174,
+                "city": "Cork",
+                "country": "IE"
+            }
+        },
+        {
+            "tag": "limerick",
+            "meta": {
+                "latitude": 52.668746,
+                "longitude": -8.627809,
+                "city": "Limerick",
+                "country": "IE"
+            }
+        },
+        {
+            "tag": "galway",
```

Comment:
I don't like tying the tag of the system with the functionality of the implementation. Please use something else, like `system_tag` or `site_tag`.

:+1: on the tags with the city name, this is the safest assumption.